### PR TITLE
add example project for create-react-app project

### PR DESCRIPTION
**Pull request by dwebfan**
Signed-off-by: Dweb Fan <dwebfan@gmail.com>

All credits are from https://github.com/antelle/argon2-browser/issues/38. Hope to use this example to save new-comer time.

### CODE REVIEW

**Comment by dwebfan** [comment]
@antelle This is still broken with my latest comment for the issue. Please let me know where I can change to solve the issue.

**Comment by dwebfan** [comment]
Pushed with new working version, and with one simple web page to show the usage.

File: examples/react/src/App.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import "./App.css";
+const argon2 = require('argon2-browser')
+
+String.prototype.hexEncode = function(){
```
Comment (antelle) [inline examples/react/src/App.js:5]: Bad practice: don't extend prototypes, delete this code from the example. The example also doesn't need to encode as hex at all, so let's remove this function.

File: examples/react/src/registerServiceWorker.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+// In production, we register a service worker to serve assets from local cache.
```
Comment (antelle) [inline examples/react/src/registerServiceWorker.js:1]: Let's delete this file, it's not needed for understanding

**Review by antelle — CHANGES_REQUESTED** [review]
Hi! 🎁 
Thanks for the example, added some comments about unnecessary code there.

Don't worry about the CI, it's because bintray is not anymore, I'll make it publish the package from GitHub actions.

File: examples/react/src/App.js (hunk 1 above)
Comment (dwebfan) [reply examples/react/src/App.js]: done.

File: examples/react/src/registerServiceWorker.js (hunk 2 above)
Comment (dwebfan) [reply examples/react/src/registerServiceWorker.js]: done.

**Comment by antelle** [comment]
Thank you very much!

**Comment by antelle** [comment]
You can also add a link to your example in the readme, or I'll do it later.

**Pull request merged**